.github/workflows/run_tests(36).yml

```diff
@@ -0,0 +1,30 @@
+name: Run tests (3.6)
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Run Tests
+      run: |
+        ./run_tests.sh
```

Comment:
dafuq? opravdu se to musi delat takhle blbe, ze xkrat zkopirujes stejny soubor? to si nemyslim...

Reply:
Asi ne, to jsem udělal rychle abych nezapoměl že to chci udělat, měl jsem v plánu to ještě předělat než ti to pošlu, ta změna z draftu byla předtím než jsem tohle přidal

Reply:
Spojené do pole, a nastavení aby se tak workflow jmenoval s proměnnými

Reply:
tohle vypada mnohem lepsi, jestli to funguje

Reply:
Mělo by, viz akce u tohohle posledního pushe